cleanup/artifactCleanup/artifactCleanup.groovy

```diff
@@ -218,3 +235,7 @@ private def getSkippedPaths(String[] repos) {
     log.info "Elapsed time to retrieve paths to skip: " + duration
     return skip
 }
+
+private def checkName(keepRelease, releaseRegex, artifactName) {
```

Comment:
So if I set `keepRelease` to `true` than the regex check is still evaluated and my release artifact might still get deleted